tests/test_sharded_zarr_store_v2.py

```
@@ -0,0 +1,906 @@
+            raise TypeError(f"Expected CID, got {type(cid).__name__}")
+        return str(decoded.encode("base32"))
+
+    async def save(self, data: bytes, codec: ContentAddressedStore.CodecInput) -> CID:
+        digest = multihash.digest(data, "sha2-256")
+        cid = CID("base32", 1, codec, digest)
+        self.store[self._key(cid)] = data
+        return cid
+
+    async def load(
+        self,
+        id: IPLDKind,
```

Ruff (0.15.20)

[error] Function argument `id` is shadowing a Python builtin

(A002)